src/PhpParser/Node/Value/ValueResolver.php

```diff
@@ -62,7 +62,7 @@ public function getStringValue(Expr $expr): string
     }
 
     /**
-     * @return mixed|null
+     * @phpstan-return mixed|null
```

Comment:
what happen with `@return mixed|null` ? why it originally deleted with `@return mixed|null` ?

Reply:
It did not, I replaced it manually. Using `@phpstan-return` instead of `@return` makes sure that the `UnionTypesRector` will not misfire and remove the annotation entirely, because it thinks it can since it considers it equivalent to `?array`.